src/legacy/core_plugins/kibana/public/dashboard/saved_dashboard/saved_dashboard.d.ts

```diff
@@ -17,9 +17,8 @@
  * under the License.
  */
 
-import { SearchSource } from 'ui/courier';
+import { SearchSourceContract } from 'ui/courier';
```

Comment:
changing `ui/` imports to relative ones will make it easier once you move courier to a different location as editor will be able to assist with updating imports